script/cli/help.lua

```diff
@@ -0,0 +1,158 @@
+        ]],
+        example = [[--doc_out_path=C:/Users/Me/Documents/myLuaProjectDocumentation]]
+    },
+    ['--logpath'] = {
+        type = 'string',
+        description = [[
+            Where the log should be written to.
+        ]],
+        default = './log',
+        example = [[--logpath=D:/luaServer/logs]]
+    },
+    ['--loglevel'] = {
+        type = 'string',
+        description = [[
+            The minimum level of logging that should appear in the logfile.
+            Can be used to log more detailed info for debugging and error reporting.
+
+            Options:
+
+                - error
+                - warn
+                - info
+                - debug
+                - trace
+        ]],
+        example = [[--loglevel=trace]]
+    },
+    ['--metapath'] = {
+        type = 'string',
+        description = [[
+            Where the standard Lua library definition files should be generated to.
+        ]],
+        default = './meta',
+        example = [[--metapath=D:/sumnekoLua/metaDefintions]]
+    },
+    ['--locale'] = {
+        type = 'string',
+        description = [[
+            The language to use. Defaults to en-us.
+            Options can be found in locale/ .
+        ]],
+        example = [[--locale=zh-cn]]
+    },
+    ['--configpath'] = {
+        type = 'string',
+        description = [[
+            The location of the configuration file that will be loaded.
+            Can be relative to the workspace.
+            When provided, config files from elsewhere (such as from VS Code) will no longer be loaded.
+        ]],
+        example = [[--configpath=sumnekoLuaConfig.lua]]
+    },
+    ['--force-accept-workspace'] = {
+        type = 'boolean',
+        description = [[
+            Allows the use of root/home directory as the workspace.
+        ]]
+    },
+    ['--socket'] = {
+        type = 'number',
+        description = [[
+            Will communicate to a client over the specified TCP port instead of through stdio.
+        ]],
+        example = [[--socket=5050]]
+    },
+    ['--develop'] = {
+        type = 'boolean',
+        description = [[
+            Enables development mode. This allows plugins to write to the logpath.
+        ]]
+    }
+}
+
+for nm, attrs in pairs(args) do
+    if attrs.type == 'string' then
+        print(nm .. "=<value>")
+    else
+        print(nm)
+    end
+    if attrs.description then
+        local normalized_description = attrs.description:gsub("^%s+", ""):gsub("\n%s+", "\n"):gsub("%s+$", "")
+        print("\n    " .. normalized_description:gsub('\n', '\n    '))
+    end
+    if attrs.type then
+        print("\n    Type: " .. (type(attrs.type) == "table" and table.concat(attrs.type, ", ") or attrs.type))
+    end
+    if attrs.default then
+        print("\n    Default: " .. tostring(attrs.default))
+    end
+    if attrs.example then
+        print("\n    Example: " .. attrs.example)
+    end
+    print()
```

Comment:
why print empty

Reply:
Just a linebreak to make the output easier to read.